### PR TITLE
Update dependencies, fix breaking change in gocql

### PR DESCRIPTION
**Pull request by Willyham**

### CODE REVIEW

File: connectors/yarpc/yarpc_test.go [hunk 1]
```diff
@@ -812,12 +812,12 @@ func TestConnector_Scan(t *testing.T) {
 			},
 			NextToken: &responseToken,
 		}, nil)
-	// failed call, return error
```
Comment (klnusbaum) [inline connectors/yarpc/yarpc_test.go]: Why do these need to be changed?

**Comment by klnusbaum** [comment]
Overall I really like this change. I just wanna understand a little better why we need to change the yarpc test. That seems orthogonal to all this.

**Comment by klnusbaum** [comment]
This is a good fix. Please update the `changelog` file as well. We'll put this in a patch version. so please create a 2.3.1 entry in the change log file.

**Comment by eculver** [comment]
my vote is for fixing gocql/gocql (by reverting ab2b96a) -- this becomes a no-op and would benefit the greater gocql community

**Comment by eculver** [comment]
This: https://github.com/gocql/gocql/pull/1002

**Comment by klnusbaum** [comment]
I don't think we need this any more thanks to @eculver 's fix. We're actually probably going to move the cassandra connector to it's own repository in a bit in order to mitigate this issue all together. So I don't think we need this.

**Pull request closed without merging**